all/src/test/java/io/opentelemetry/all/FallbackArtifactsTest.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.all;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class FallbackArtifactsTest {
+
+  @Test
+  void exporterJaegerProto() {
+    classAvailable(io.opentelemetry.exporter.jaeger.proto.api_v2.Collector.class);
+    classAvailable(io.opentelemetry.exporter.jaeger.proto.api_v2.CollectorServiceGrpc.class);
+    classAvailable(io.opentelemetry.exporter.jaeger.proto.api_v2.Model.class);
+  }
+
+  private static void classAvailable(Class<?> clazz) {
+    Assertions.assertThatCode(() -> Class.forName(clazz.getName())).doesNotThrowAnyException();
```

Comment:
By this point `clazz` is already loaded, and this will never throw. Should we pass the class name using a string?

Reply:
Yea that's true. Let's go with that.